### DISCUSSION TITLE
質問のトレンド・新着が怪しい英語の質問だらけになっている

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#414] [ヘルプ] [answered]
1/10に、BOTで投稿されたような英語の質問が大量に投稿されています。さすがに多すぎて個別に報告していられないので、こちらに記載させていただきます。軽くみた感じでは、外部サイトのリンクは無かったようですが、危険なサイトへ誘導されるといった事も考えれますので、何か対策をしていただけませんでしょうか？

![bot](https://user-images.githubusercontent.com/12453432/211698076-343c1739-9e4e-4811-ae55-3cf40e123b65.png)

**Comment by tanaka-asami415** [comment, marked as answer]
@radian-jp 
ご報告ありがとうございます。ご不便をおかけして申し訳ありません🙇

先ほど、こちらのDiscussion https://github.com/increments/qiita-discussions/discussions/415 でもアナウンスいたしましたが
現在、判定を強化するための対策を行っておりますので、もうしばらくご容赦いただけますと幸いです🙇

**Reply by tanaka-asami415** [reply]
@radian-jp 
先日はご報告いただきありがとうございました🙇

全てを防ぎ切れてはおりませんが、スパム判定を強化し、
一旦の収束が見られたため本件クローズさせていただきます。引き続き対策をしていきます。
